src/components/Garden/Feed/ProposalCreator.js

```diff
@@ -16,7 +16,7 @@ import { ZERO_ADDR } from '@/constants'
 
 const addressCache = new Map()
 
-function ProposalCreator({ proposal }) {
+function ProposalCreator({ proposal, onViewProfile }) {
```

Comment:
Maybe convert that ProposalCreator.js file to TS too?

Reply:
I did convert it to `tsx`, but notice that we have an icon from `1hive-ui`, and it's giving error in TS. The icon is `EthIdenticon`

Reply:
Maybe @reznord can give more context on what could be the reason for that error?